### PR TITLE
Add belly-button linting

### PR DESCRIPTION
**Pull request by cjihrig**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 [![Build Status](https://travis-ci.org/continuationlabs/big-time.svg?branch=master)](https://travis-ci.org/continuationlabs/big-time)
 [![npm](https://img.shields.io/npm/v/big-time.svg)](https://www.npmjs.com/package/big-time)
 
+[![belly-button-style](https://cdn.rawgit.com/continuationlabs/belly-button/master/badge.svg)](https://github.com/continuationlabs/belly-button)
```
Comment (arb) [inline README.md]: :clap:

File: src/index.js [hunk 2]
```diff
@@ -1,43 +1,38 @@
...
+  close () {
+    clearTimeout(this._timeout);
+  }
 }
 
-const _setTimeout = (...args) => new Timeout(...args);
+const _setTimeout = (...args) => { return new Timeout(...args); };
 
-const _clearTimeout = (timer) => timer.close();
+const _clearTimeout = (timer) => { return timer.close(); };
```
Comment (arb) [inline src/index.js]: You don't actually need the `return` here.

**Comment by arb** [comment]
Do you want to try scrabble out here? This is a decent candidate since it already is transpiled.

**Comment by cjihrig** [comment]
`scrabel` is your call. I'd certainly be open to it.

**Comment by arb** [comment]
Let's do a major bump and include scrable. I know good-file uses this and I wouldn't want to break that.

File: src/index.js (hunk 2 above)
Comment (cjihrig) [inline src/index.js]: Does `timer.close()` return anything?

**Comment by cjihrig** [comment]
I'd say let's get this change in and do a patch release. Then, do the scrabel major bump separately. I'll create an issue for it.

**Pull request merged**